tests.test.ts

```diff
@@ -0,0 +1,67 @@
+import { api } from "./convex/_generated/api";
+import { ConvexReactClient } from "convex/react";
+import WebSocket from "ws";
+import { UserIdentity } from "convex/server";
+import { ConvexError } from "convex/values";
+
+
+class IdentityFactory {
+  private _nextSubjectId: number = 0;
+  constructor() {}
+
+  newIdentity(args: Partial<Omit<UserIdentity, "tokenIdentifier">>): Omit<UserIdentity, "tokenIdentifier"> {
+    const subject = `test subject ${this._nextSubjectId}`;
+    this._nextSubjectId += 1;
+    const issuer = "test issuer"
+    return {
+      ...args,
+      subject,
+      issuer
+    }
+  }
+}
+
+function setAuth(client: ConvexReactClient, identity: Omit<UserIdentity, "tokenIdentifier">) {
+  const ADMIN_KEY = '0135d8598650f8f5cb0f30c34ec2e2bb62793bc28717c8eb6fb577996d50be5f4281b59181095065c5d0f86a2c31ddbe9b597ec62b47ded69782cd';
```

Comment:
How do people get this, and how should they configure it? Env var (node.js one)?

Reply:
I think the plan is just to bake this into all local backends (I copied it from the Justfile from that repo)